IHP/Server.hs

```diff
@@ -62,7 +62,9 @@ run configBuilder = do
                                 methodOverridePost $
                                     application
 
-    run `finally` (frameworkConfig |> get #logger |> get #cleanup)
+    run `finally` do
+        frameworkConfig |> get #logger |> get #cleanup
+        readIORef autoRefreshServer >>= (\autoRefreshServer -> autoRefreshServer |> get #subscriptions |> mapM_ Async.uninterruptibleCancel)
```

Comment:
Can we extract this into a `stopAutoRefreshServer` function? Otherwise we're kind of leaking implementation details of the auto refresh server in the `IHP.Server` module